src/main/java/org/alliancegenome/curation_api/model/entities/GenomicEntity.java

```diff
@@ -15,6 +17,15 @@
 
 import lombok.*;
 
+@JsonTypeInfo(
```

Comment:
@kthorat-prog I don't think we want this at both levels?

Reply:
So shall I remove it from genomic Entity?

Reply:
Was it solving an issue? I think it depends on what the subject field in the disease annotation is pointing to.

Reply:
So Gene, Allele, and AGM are all sub classes of GenomicEntity so I would guess we would want it at that level.

Reply:
Yes. I think so too.

Reply:
I have removed type from BiologicalEntity class .Please check.